scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -138,14 +150,15 @@ def build_graph(df, dataset):
         df = df.Define("weight_fullMuonSF_withTrackingReco", muon_efficiency_helper, ["goodMuons_pt0", "goodMuons_eta0", "goodMuons_charge0", "passIso"])
         df = df.Define("weight_newMuonPrefiringSF", muon_prefiring_helper, ["Muon_correctedEta", "Muon_correctedPt", "Muon_correctedPhi", "Muon_looseId"])
 
-        applyScetlibCorr = True
```

Comment:
I don't understand if you actually use the noScetlibCorr argument. Shouldn't it be used here? There is also a conflict that I introduced (sorry) related to these lines of code

Reply:
It was changed to a command line option, anyway the conflict should be fixed now